### DISCUSSION TITLE
Ruckeln Standort Pfeil

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#131] [Q&A] [answered]
Habe heute das Routing bei einer Testfahrt Probiert. Funktioniert sehr gut ! Auch das zoomen vor Kreuzungen funktionierte hervorragend.
Eine Sache ist mir aufgefallen. Beim navigieren ruckelt der Pfeil etwas. Dauert 2 Sekunden dann wird der Pfeil bewegt. Das ganze wiederholt sich ständig. Das heißt der Pfeil bewegt sich nicht kontinuierlich.

Ist nicht wirklich ein großes Problem da der Standort immer stimmt. Liegt es vielleicht an meinem Telefon das es zu langsam rechnet. ( Samsung S 8  )
Wie gesagt wenn sich das nicht lösen lässt kann ich damit leben.
LG

**Comment by devemux86** [comment]
@1maxi2 thanks for the report!

What have you selected in "Settings | Location | Location service"?
Are you running another application that uses GPS location?

Have you enabled the "Settings | Navigation | Advanced | Snap to route"?
To display GPS location in real time, please do not enable this option.

Animations result in slow location updates and consume battery.
You can try the "Settings | Location | Animated location" option.

---

@1maxi2 danke für den Bericht!

Was haben Sie unter "Einstellungen | Standort | Standortservice" ausgewählt?
Führen Sie eine andere Anwendung aus, die GPS-Standort verwendet?

Haben Sie die Option "Einstellungen | Navigation | Erweitert | Position auf Route" aktiviert?
Um den GPS-Standort in Echtzeit anzuzeigen, aktivieren Sie diese Option bitte nicht.

Animationen führen zu langsamen Standortaktualisierungen und verbrauchen Akku.
Sie können die Option "Einstellungen | Standort | Animierter Standort" ausprobieren.

**Reply by 1maxi2** [reply]
Standort zulassen: Cruiser, Wetter online, Maps, Google Play,  Kurviger Pro,  Kurviger.

Position auf Route ist nicht aktiviert

Animierter Standort ist nicht aktiviert.

Vielleicht ist das " ruckeln " normal.
Wäre interessant was andere Benutzer darüber berichten.

**Reply by devemux86** [reply]
> Standort zulassen: Cruiser, Wetter online, Maps, Google Play, Kurviger Pro, Kurviger.

I meant, are other applications that use the GPS running at the same time as Cruiser?

Ich meinte, ob andere Anwendungen, die das GPS verwenden, gleichzeitig mit Cruiser laufen?

**Reply by 1maxi2** [reply]
Nein während der Navigation läuft nur Cruiser.
Vielleicht ist das Verhalten normal.
Gibt es irgendwo ein Video welches den Standort Pfeil während der Navigation zeigt ?

**Reply by devemux86** [reply]
- Can you try another option under "Settings | Location | Location service"?
- You can also enable the "Settings | Location | Animated location" option.
- Or "Settings | Location | Advanced | Kalman filter".

---

- Können Sie eine andere Option unter "Einstellungen | Standort | Standortservice" ausprobieren?
- Sie können auch die Option "Einstellungen | Standort | Animierter Standort" aktivieren.
- Oder "Einstellungen | Standort | Erweitert | Kalman-Filter".

**Reply by 1maxi2** [reply]
O.k. Danke, werde bei der nächsten Ausfahrt testen ob es eine Änderung gibt.

**Reply by devemux86** [reply]
@1maxi2 

I recommend to not try all options together, but one option separately each time.

Ich empfehle, nicht alle Optionen zusammen auszuprobieren, sondern jedes Mal eine Option separat.

**Reply by 1maxi2** [reply]
O.k.

**Reply by WolfgangZ42** [reply]
> Gibt es irgendwo ein Video welches den Standort Pfeil während der Navigation zeigt ?


https://user-images.githubusercontent.com/121806238/232561691-eefa7f2f-ba16-4f38-bdac-9994c18614dc.mp4

**Reply by 1maxi2** [reply]
Danke Wolfgang für das Video ! Bei dir läuft der Pfeil perfekt kontinuierlich.
Bei mir ruckelt das auf alle Fälle.
Werde jetzt die Tipps von devemux ausprobieren. Sollte das nicht helfen werde ich versuchen ein Video einzustellen wie das bei mir aussieht.

**Reply by 1maxi2** [reply, marked as answer]
Danke für die Hilfestellung. " Animierter Standort " aktivieren war die Lösung ! Jetzt bewegt sich der Pfeil flüssig so wie im Video von Wolfgang. 😊
